src/service-module/service-module.getters.ts

```diff
@@ -73,6 +73,9 @@ export default function makeServiceGetters() {
         data: values
       }
     },
+    count: (state, getters) => params => {
+      return getters.find(state)(params).total
```

Comment:
```suggestion
      return getters.find(params).total
```

Reply:
I think this is being called inappropriately.

Furthermore, the `find` getter is going to do some extra stuff that is unnecessary for `count` if someone puts certain keywords in the query. I'm a fan of keeping this simple like it is because the gains are probably negligible (unless someone accidentally puts a `$sort` into a `count` on a list with thousands of items).

We could have this `count` getter delete `$sort`, `$skip`, `$limit`, and `$select` from the query just to expedite the `find` getter.

Reply:
Thanks for your suggestion and thoughts. Your suggestion does not work. It needs to be this way.

I'm definitly on your side! I omitted the filters. Now it just passes a `cleanQuery` to the `find`-getter.